aea/configurations/schemas/skill-config_schema.json

```diff
@@ -146,36 +125,9 @@
         },
         "args": {
           "type": "object"
-        }
-      }
-    },
-    "handler": {
-      "type": "object",
-      "additionalProperties": false,
-      "required": [
-        "class_name"
-      ],
-      "properties": {
-        "class_name": {
-          "type": "string"
         },
-        "args": {
-          "type": "object"
-        }
-      }
-    },
-    "model": {
-      "type": "object",
-      "additionalProperties": false,
-      "required": [
-        "class_name"
-      ],
-      "properties": {
-        "class_name": {
+        "file_path": {
           "type": "string"
-        },
-        "args": {
-          "type": "object"
         }
       }
     }
```

Comment:
the diffs are quite ugly, but the idea is that:
- "handler", "behaviour" and "model" schemas are squashed into one schema, "skill_component_configuration"
- "handlers", "behaviours" and "models" schemas are squashed into one schema, "skill_component_list".

Reply:
no reason to keep them separated.